Assets/Scripts/Missions/Mission.cs

```diff
@@ -4,6 +4,8 @@
 
 public class AMission : MonoBehaviour
```

Comment:
My only nitpick: why is this class called "AMission" if the filename is "Mission.cs"?